apisix/cli/ops.lua

```diff
@@ -189,6 +189,13 @@ local function init(env)
        and #allow_admin == 1 and allow_admin[1] == "127.0.0.0/24" then
         checked_admin_key = true
     end
+    -- check if admin_key is required
```

Comment:
It seems that this logic is not necessary, It's just for start APISIX, but you can not access apisix without admin key, right?
```
    if yaml_conf.apisix.enable_admin and allow_admin
       and #allow_admin == 1 and allow_admin[1] == "127.0.0.0/24" then
        checked_admin_key = true
    end
```

Reply:
IMHO These codes does not conflict with the current feature.

This logic is to allow `127.0.0.0/24` authentication-free access to the Admin API by default.

Reply:
If this code is useless, would you mind creating a PR to remove it?

Reply:
> OK, it's just for starting, righting?
Yes. It just a check before starting.

> if set `127.0.0.0/24`, the user cann't access the APISIX? I don't find the code
I mean if set `127.0.0.0/24`, the user **CAN** access the APISIX with empty admin key configuraion. Otherwise the APISIX will start failed (such as `0.0.0.0/0` in allow_admin but empty admin_key)..

Reply:
It is a default behavior that allow the user access Admin API from local.

Reply:
I checked this, after set `127.0.0.0/24` and remove the admin key:
```
curl http://127.0.0.1:9180/apisix/admin/upstreams/1 -X PUT -d '
{
    "type": "chash",
    "key": "remote_addr",
    "nodes": {
        "127.0.0.1:80": 1,
        "foo.com:80": 2
    }
}'
{"error_msg":"failed to check token"}
```

Reply:
We talk about this in another PR